Tests/FoundationInternationalizationTests/Formatting/ICUPatternGeneratorTests.swift

```diff
@@ -11,17 +11,17 @@ import TestSupport
 #endif
 
 #if canImport(FoundationInternationalization)
-@testable import FoundationEssentials
-@testable import FoundationInternationalization
+import FoundationEssentials
+import FoundationInternationalization
 #endif
 
 #if FOUNDATION_FRAMEWORK
-@testable import Foundation
+import Foundation
 #endif
 
 final class ICUPatternGeneratorTests: XCTestCase {
 
-    typealias DateFieldCollection = Date.FormatStyle.DateFieldCollection
+    package typealias DateFieldCollection = Date.FormatStyle.DateFieldCollection
```

Comment:
Is this needed? The test module isn't a dependency of any other module, so it shouldn't need anything more permissive than `internal`

Reply:
Yes, I changed this by mistake